Dockerfile

```diff
@@ -24,9 +24,5 @@ ENV NODE_OPTIONS="--max_old_space_size=4096"
 # Listen / accept connections from all IP addresses.
 # NOTE: At this time it is only possible to run Docker container in Production mode
 # if you have a public URL. See https://github.com/DSpace/dspace-angular/issues/1485
-ENV NODE_ENV=production
```

Comment:
nonono we want keep this

Reply:
but pavol said that this dockerfile should not be used in production because that's for the dockerfile.dict is for. but if you want it I will use production instead of development